config/dependency-injection/services.php

```diff
@@ -45,6 +49,9 @@
 	'surfaces/values',
 	'presenters',
 	'config/migrations',
+	'exceptions/oauth',
```

Comment:
I'd exclude the exceptions and values directories entirely rather than only subdirectories.

Reply:
The values directory has other value objects as well that seem dependent on the auto-wiring.